rest-servlet/src/main/java/org/hawkular/inventory/rest/RestEnvironments.java

```diff
@@ -57,14 +60,16 @@
 
     @GET
     @Path("/{tenantId}/environments")
-    @ApiOperation("Returns all environments under given tenant.")
+    @ApiOperation("Returns all environments under given tenant. Accepts paging query parameters.")
```

Comment:
On 28 Apr 2015, at 11:51, Jirka Kremser wrote:

>> @@ -57,14 +60,16 @@
>>
>>   @GET
>>   @Path("/{tenantId}/environments")
>> -    @ApiOperation("Returns all environments under given tenant.")
>> +    @ApiOperation("Returns all environments under given tenant. 
>> Accepts paging query parameters.")
>
> What about `@QueryParam`s instead of extracting it from `UriInfo`?

While we use that approach in RHQ, I see the benefit of not constantly 
repeating
ourselves via the solution from @metlos